src/app/child-dev-project/children/model/genders.ts

```diff
@@ -13,4 +13,8 @@ export const genders: ConfigurableEnumValue[] = [
     id: "F",
     label: $localize`:Label gender:female`,
   },
+  {
+    id: "X",
+    label: $localize`:Label gender:X`,
```

Comment:
Could you set the label to something more self explaning (e.g. `"Non-binary/third gender"` see [comment](https://github.com/Aam-Digital/ndb-core/issues/965#issuecomment-916415286)). And also provide a german translation for it?

Reply:
Yes. I will do it now!